doc/manual/tags.md

```diff
@@ -3,96 +3,296 @@
 ## Base package tags
 
 * Name (1000) string
```

Comment:
I wonder what is the number in parentheses. Probably some internal ID, but it would be nice to have it mentioned somewhere.

Reply:
Actually, there is some explanation in https://github.com/rpm-software-management/rpm/blob/master/doc/manual/queryformat.md#tags if I understand this, so this could be interlinked.

Reply:
This is about the package format, not spec tags. The numbers are the tag ID's that the physical headers have, the names are entirely for human convenience.